tcms_api/xmlrpc.py

```diff
@@ -114,7 +116,14 @@ class TCMSXmlrpc:
     session_cookie_name = "sessionid"
     transport = None
 
-    def __init__(self, username, password, url):
+    def __init__(
+        self,
+        username,
+        password,
+        url,
+        allow_unverified_ssl: bool = False,
+        extra_headers: list[tuple[str, str]] = [],
+    ):
```

Comment:
Will have to define these for the inherited class TCMSKerbXmlrpc, and take care to pass them down appropriately otherwise we have broken method signatures in the inheritance chain.